src/payment.js

```diff
@@ -56,16 +29,16 @@ function Payment (wallet, payment) {
     from: null, // origin
     amounts: [], // list of amounts to spend entered in the form
     to: [], // list of destinations entered in the form
-    feePerKb: serverFeeFallback.default.fee, // default fee-per-kb used in basic send
+    feePerKb: serverFeeFallback.legacyCapped, // default fee-per-kb used in basic send
     extraFeeConsumption: 0, // if there is change consumption to fee will be reflected here
     sweepFee: 0,  // computed fee to sweep an account in basic send (depends on fee-per-kb)
     sweepAmount: 0, // computed max spendable amount depending on fee-per-kb
     balance: 0, // sum of all unspents values with any filtering     [ payment.sumOfCoins ]
     finalFee: 0, // final absolute fee that it is going to be used no matter how was obtained (advanced or regular send)
     changeAmount: 0, // final change
-    absoluteFeeBounds: [0, 0, 0, 0, 0, 0], // fee bounds (absolute) per fixed amount
-    sweepFees: [0, 0, 0, 0, 0, 0], // sweep absolute fee per each fee per kb (1, 2, 3, 4, 5, 6)
-    maxSpendableAmounts: [0, 0, 0, 0, 0, 0],  // max amount per each fee-per-kb
+    absoluteFeeBounds: [0, 0, 0], // fee bounds (absolute) per fixed amount
+    sweepFees: [0, 0, 0], // sweep absolute fee per each fee per kb (slow, regular, priority)
+    maxSpendableAmounts: [0, 0, 0],  // max amount per each fee-per-kb
```

Comment:
I think it would be better to change these arrays to objects of `{ slow, regular, priority }`, `payment.fees` is an object now so the order is implied and probably shouldn't be relied on.